tools/repositories.bzl

```diff
@@ -16,6 +16,7 @@ load("//tools:bazel_hash_dict.bzl", "BAZEL_HASH_DICT")
 load(":common.bzl", "BAZEL_VERSIONS")
 
 _BAZEL_BINARY_PACKAGE = "http://releases.bazel.build/{version}/release/bazel-{version}{installer}-{platform}.{extension}"
```

Comment:
Could you convert this to a list of strings and then map the .format operation over the list to create the url list?

Reply:
👍 good call